drivers/wifi/nrf_wifi/Kconfig.nrfwifi

```
@@ -524,12 +524,28 @@ config NRF70_RSSI_STALE_TIMEOUT_MS
 	  value as the driver does not store it and requires RPU to provide the
 	  information.
 
+config NRF_WIFI_GLOBAL_HEAP
+	bool "Use K_HEAP for Wi-Fi driver"
+	depends on KERNEL_MEM_POOL && ((HEAP_MEM_POOL_SIZE > 0) || !HEAP_MEM_POOL_IGNORE_MIN)
```

Either heap is configured or ignore_min is set, no?

```suggestion
	depends on KERNEL_MEM_POOL && ((HEAP_MEM_POOL_SIZE > 0) || HEAP_MEM_POOL_IGNORE_MIN)
```

---

Sure, I would say that if a customer is configuring that way and uses `HEAP_MEM_POOL_IGNORE_MIN` then that's expected, I don't think we can handle such cases, e.g., if it's non-zero but  low.

---

Done